android/shopify_buy_plugin/build.gradle

```diff
@@ -28,6 +28,7 @@ android {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile group: 'com.google.code.gson', name: 'gson', version: '2.3.1'
```

Comment:
Hmm I just realized that I don't think JSONObject is mocked when running unit tests.